pkg/subnet/public.go

```diff
@@ -530,6 +559,44 @@ func (d *PublicDeployer) createTransformSubnetTX(
 	return &tx, nil
 }
 
+func (d *PublicDeployer) issueAddPermissionlessValidatorTX(
+	recipientAddr ids.ShortID,
+	stakeAmount uint64,
+	subnetID ids.ID,
+	nodeID ids.NodeID,
+	assetID ids.ID,
+	startTime uint64,
+	endTime uint64,
+	wallet primary.Wallet,
+) (ids.ID, error) {
+	options := d.getMultisigTxOptions([]ids.ShortID{})
```

Comment:
mmm probably we need to be more clear on this, as there is no multisig involved here. dont? maybe change the function name

Reply:
I should make two function now, keep the getMultisigTxOptions but make it to call another one just to set the change owner, and also use that function here